app/templates/components/file-picker.hbs

```diff
@@ -11,9 +11,15 @@
 {{/if}}
 
 {{#if progress}}
-  <div class="file-picker__progress">
-    <span class="file-picker__progress__value" style={{progressStyle}}></span>
-  </div>
+  {{#if showProgress}}
+    {{#if isProgressSupported}}
+      <progress value={{progressValue}} max="100">{{progress}} %</progress>
```

Comment:
What for? Element could be easily selected by `.file-picker progress`. I think mostly defaults will be used as they are good integrated in browser and OS style. Cause default styles of progress and div element are very different existing styles have to be adjusted. So in my opinion using the same class won't be a help for migration.